__tests__/app.test.js

```diff
@@ -147,4 +147,46 @@ describe('GET - /api/reviews/:review_id/comments', () => {
                 expect(body.comments).toEqual([])
             })
     });
+});
+
+describe('POST - /api/reviews/:review_id/comments', () => {
+    it('201: POST response with adding new object into database and sending out message', () => {
+        const newComment = { username: 'bainesface', body: "Gaming is GOOD" }
+        return request(app)
+            .post("/api/reviews/2/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                const { comment } = body
+                expect(comment.body).toBe("Gaming is GOOD");
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    body: "Gaming is GOOD",
+                    votes: expect.any(Number),
+                    author: expect.any(String),
+                    review_id: 2,
+                    created_at: expect.any(String)
+                });
+            })
+    });
+    it('404: POST response with error message when input a invalid review_id', () => {
+        const newComment = { username: 'bainesface', body: "Gaming is GOOD" };
+        return request(app)
+            .post("/api/reviews/999/comments")
+            .send(newComment)
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("resource not exist");
+            })
+    });
+    it('404: POST response with error message when input a invalid username', () => {
+        const newComment = { username: 'aaa', body: "Gaming is GOOD" };
+        return request(app)
+            .post("/api/reviews/2/comments")
+            .send(newComment)
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("resource not exist");
+            })
+    });
```

Comment:
Just missing two more tests:

- [ ] 400 - invalid article id
- [ ]  400 missing required field in the post body object